hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -633,6 +657,9 @@ def create_volume(self, volume):
             extras = {'comment': json.dumps(comments),
                       'tpvv': tpvv, }
 
+            if len(self.config.hpe3par_snapcpg):
+                extras['snapCPG'] = self.config.hpe3par_snapcpg[0]
+
```

Comment:
if the config param for snapCPG is not given (in hpe.conf) can we default the snapcpg to the actual cpg where the volume will be created ?

Reply:
Okay.